go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -113,12 +113,11 @@ func (oc *Controller) syncEgressFirewall(egressFirwalls []interface{}) {
 	// update the direction of each egressFirewallACL if needed
 	if len(egressFirewallACLs) != 0 {
 		opModels := []libovsdbops.OperationModel{}
-		for i, egressFirewallACL := range egressFirewallACLs {
-			aclName := egressFirewallACLs[i].Name
+		for i := range egressFirewallACLs {
+			egressFirewallACL := egressFirewallACLs[i]
 			egressFirewallACL.Direction = types.DirectionToLPort
 			opModels = append(opModels, libovsdbops.OperationModel{
-				Model:          &egressFirewallACL,
-				ModelPredicate: func(acl *nbdb.ACL) bool { return acl.Name == aclName },
+				Model: &egressFirewallACL,
```

Comment:
What about `BulkOp=true` here? I don't think we need it since we shouldn't be returning multiple models in each Op (